dataservice/api/diagnosis/models.py

```diff
@@ -16,6 +16,9 @@ class Diagnosis(db.Model, Base):
     :param tumor_location: Location of the tumor
     :param age_at_event_days: Age at the time of diagnosis expressed
     in number of days since birth.
+    :param mondo: An ontology that harmonizes diagnosis
```

Comment:
Are these supposed to be id fields that reference the ontology term (like HPO)? If so, we should probably name them `ncit_id`, `mondo_id`